WordPress/Classes/Utility/OptimizelyHelper.swift

```diff
@@ -29,8 +29,22 @@ internal var optimizelyEnableNewSigninFlowKey: OptimizelyVariableKey = Optimizel
         Optimizely.disableSwizzle() // Disable's the Optimizely visual editor.
         Optimizely.sharedInstance().disableGesture = true
         preregisterOptimizelyKeys()
-        Optimizely.startOptimizelyWithAPIToken(ApiCredentials.optimizelyAPIKey(), launchOptions: launchOptions)
-        Optimizely.refreshExperiments()
+
+        // Check for a signed in account, or at least one blog added to the app.
+        // Load Optimizely asych in these cases as we won't be showing the sign-in
+        // screen right away.
+        let blogService = BlogService(managedObjectContext: ContextManager.sharedInstance().mainContext)
+        blogService.blogCountForAllAccounts()
```

Comment:
Ugh. No. That's cruft from me fighting with flaky syntax highlighting that I meant to clean up.  Will remove.